tests/keras/backend/backend_test.py

```diff
@@ -1689,6 +1689,26 @@ def test_ctc_decode_greedy(self):
         assert np.alltrue(decode_pred_np == decode_pred)
         assert np.allclose(log_prob_pred_np, log_prob_pred)
 
+    @pytest.mark.skipif(K.backend() != 'tensorflow',
+                        reason='tensorflow-way slice is '
+                        'only supported in tensorflow.')
+    @pytest.mark.parametrize('npt,x_start,x_size', [
+        (np.array([[[1, 1, 1], [2, 2, 2]],
+                   [[3, 3, 3], [4, 4, 4]],
+                   [[5, 5, 5], [6, 6, 6]]]), [1, 0, 0], [1, 1, 3]),
+        (np.array([[[1, 1, 1], [2, 2, 2]],
+                   [[3, 3, 3], [4, 4, 4]],
+                   [[5, 5, 5], [6, 6, 6]]]), [1, 0, 0], [1, 2, 3]),
+        (np.array([[[1, 1, 1], [2, 2, 2]],
+                   [[3, 3, 3], [4, 4, 4]],
+                   [[5, 5, 5], [6, 6, 6]]]), [1, 0, 0], [2, 1, 3])
```

Comment:
`npt` and `x_start` are always the same, they can be declared in the test function itself. It doesn't need to be in the parameters of `parametrize`.

Reply:
Thanks for the advice.
I agree with your opinion.